lxd/device/nic_bridged.go

```diff
@@ -989,8 +984,13 @@ func (d *nicBridged) removeFilters(m deviceConfig.Device) {
 		return
 	}
 
+	allowedNetworks, err = d.allowedNetworks(IPv4Alloc.IP, IPv6Alloc.IP)
```

Comment:
FWIW I woudln't mind just calling it `allowedIPs` in this case, as it works nicely in the user facing error and the data type indicates that one can specify multiple IPs because it is an IPNet.

Reply:
The function is now called `allowedIPNets` and the variable `allowedIPs`. I can call them both `allowedIPs` if you prefer, however as the `allowedIPNets` function no longer has a receiver this causes a name collision.